port/win/env_win.cc

```diff
@@ -433,9 +435,10 @@ IOStatus WinFileSystem::OpenWritableFile(
   } else {
     // Here we want the buffer allocation to be aligned by the SSD page size
     // and to be a multiple of it
-    result->reset(new WinWritableFile(
-        fname, hFile, std::max(GetSectorSize(fname), GetPageSize()),
-        c_BufferCapacity, local_options));
+    size_t sector_size = GetSectorSize(fname);
+    result->reset(
+        new WinWritableFile(fname, hFile, std::max(sector_size, GetPageSize()),
+                            c_BufferCapacity, sector_size, local_options));
```

Comment:
I might not be following this well, but won't this potentially align based on sector rather than page size?  This might be correct, but is not what the comment indicates.

Reply:
The buffer alignment should go with either sector size or page size, depends on which one is larger. I think the comment was added originally because the it's really rare that sector size is greater.